CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 - On X11, the `Moved` event is no longer sent when the window is resized without changing position.
 - `MouseCursor` and `CursorState` now implement `Default`.
 - `WindowBuilder::with_resizable` implemented for Windows, X11, and macOS.
+- `Window::set_resizable` implemented for MacOS, X11, and Windows.
```

Comment:
The correct stylization is "macOS", and it also reads a little easier if you use the same ordering as in the previous line.